client/src/components/SignupForm/index.tsx

```diff
@@ -23,9 +27,15 @@ function SignupForm() {
   const onFinish = async (values: SignupCredentials) => {
     setLoading(true);
     try {
-      const data: SignupCredentials = await SignupSchema.validate(values, { abortEarly: false });
-      console.log(data);
+
+      const formData: SignupCredentials = await SignupSchema.validate(values, { abortEarly: false });
+
+      const { data } = await axios.post("/api/v1/auth/signup", { formData })
```

Comment:
Use a more descriptive variable name instead of 'data' to clarify the purpose of the response